test/firebase_auth_mocks_test.dart

```diff
@@ -390,6 +390,25 @@ void main() {
     expect(user.displayName, 'New Bob');
   });
 
+  test('User.updateEmail works', () async {
+    final auth = MockFirebaseAuth(signedIn: true, mockUser: tUser);
+    final user = auth.currentUser;
+    expect(
+      () async => await user!.updateEmail('john@domain.tld'),
+      returnsNormally,
+    );
+  });
+
+  test('User.updateEmail can throw exception', () async {
+    final auth = MockFirebaseAuth(signedIn: true, mockUser: tUser);
+    tUser.exception = FirebaseAuthException(code: 'invalid-email');
+    final user = auth.currentUser;
+    expect(
+      () async => await user!.updateEmail('john@domain.tld'),
```

Comment:
Can you try removing async and await here? It should still work. If it does not, feel fee to leave this as is.

See #70.